tests/cli/test_cmds_spark_run.py

```diff
@@ -123,11 +122,17 @@ class TestConfigureAndRunDockerContainer:
         'fake_dir',
     )
 
+    @pytest.fixture
+    def mock_create_spark_config_str(self):
+        with mock.patch(
+            'paasta_tools.cli.cmds.spark_run.create_spark_config_str'
```

Comment:
add `autospec=True`

Reply:
yep, good call.  TravisCI is telling me this, too :-/